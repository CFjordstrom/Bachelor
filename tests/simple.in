/[a-zA-Z0-9]*(ab(c)+){3}.|abc123?/