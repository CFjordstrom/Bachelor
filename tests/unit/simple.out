-regex
abc*1233*\+\*\?